src/Payments/Payment.php

```diff
@@ -34,39 +34,39 @@ final class Payment extends DateAwareEntity {
     private $objectId;
 
     /**
-     * @var float
+     * @var ?float
```

Comment:
Nie wydaje mi się, żeby to mogło być kiedykolwiek nullem.

Reply:
Poprawię